<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InstalarJuegoClass</class>
 <widget class="QDialog" name="InstalarJuegoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>222</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>222</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>222</height>
   </size>
  </property>
  <property name="windowTitle">
   <string> Instalar un Juego mediante DOSBox</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/grlida.qrc">
    <normaloff>:/img16/applications.png</normaloff>:/img16/applications.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Tipo de Origen:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbxMontaje_type_drive">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtOrigenPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btnOrigen">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/carpeta_0.png</normaloff>:/img16/carpeta_0.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btnOrigenClear">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1" rowspan="2">
      <widget class="QListWidget" name="lwOrigenMultiPath">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="btnSubirIso">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/go-up.png</normaloff>:/img16/go-up.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnDeleteIso">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="btnBajarIso">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/go-down.png</normaloff>:/img16/go-down.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Seccionar Destino:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtDestinoPath">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="btnDestino">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/carpeta_0.png</normaloff>:/img16/carpeta_0.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="btnDestinoClear">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>26</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/limpiar.png</normaloff>:/img16/limpiar.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Seccionar Origen:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout_1">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/cancelar.png</normaloff>:/img16/cancelar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Ejecutar DOSBox</string>
       </property>
       <property name="icon">
        <iconset resource="../res/grlida.qrc">
         <normaloff>:/img16/aplicar.png</normaloff>:/img16/aplicar.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbxMontaje_type_drive</tabstop>
  <tabstop>txtOrigenPath</tabstop>
  <tabstop>btnOrigen</tabstop>
  <tabstop>btnOrigenClear</tabstop>
  <tabstop>lwOrigenMultiPath</tabstop>
  <tabstop>btnSubirIso</tabstop>
  <tabstop>btnBajarIso</tabstop>
  <tabstop>btnDeleteIso</tabstop>
  <tabstop>txtDestinoPath</tabstop>
  <tabstop>btnDestino</tabstop>
  <tabstop>btnDestinoClear</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="../res/grlida.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>InstalarJuegoClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>214</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOrigenClear</sender>
   <signal>clicked()</signal>
   <receiver>txtOrigenPath</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOrigenClear</sender>
   <signal>clicked()</signal>
   <receiver>lwOrigenMultiPath</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>57</y>
    </hint>
    <hint type="destinationlabel">
     <x>355</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnDestinoClear</sender>
   <signal>clicked()</signal>
   <receiver>txtDestinoPath</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
